cognite/client/_api/data_modeling/views.py

```diff
@@ -99,7 +99,10 @@ def retrieve(
         """`Retrieve a single view by id. <https://developer.cognite.com/api#tag/Views/operation/byExternalIdsViews>`_
 
         Args:
-            ids (ViewIdentifier | Sequence[ViewIdentifier]): View identifier(s)
+            ids (ViewIdentifier | Sequence[ViewIdentifier]): The view identifier(s). This can be given as a tuple of
+                strings or a ViewId object. For example, ("my_space", "my_view"), ("my_space", "my_view", "my_version"),
+                or ViewId("my_space", "my_view", "my_version"). Note that version is optional, if not provided, all versions
```

Comment:
Is it better to direct the reader to the example section? (And possibly add more examples?)

Reply:
Why not both?

Reply:
Both is better ofc! 😉